src/test/scala/mesosphere/marathon/raml/VolumeConversionTest.scala

```diff
@@ -241,4 +318,81 @@ class VolumeConversionTest extends UnitTest {
       }
     }
   }
+
+  def persistentVolumeFrom(raml: PodPersistentVolume): state.PersistentVolume = {
+    Some(raml.asInstanceOf[PodVolume].fromRaml)
+      .collect { case pv: state.PersistentVolume => pv }
+      .getOrElse(fail("expected PersistentVolume"))
+  }
+
+  "test pod volume disk types" when {
```

Comment:
I'd be tempted to use a table and this totally works

Reply:
You are so right @timcharper – I never used tables before, I now love tables :D